experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card class="statuses-details">
+    <ul class="breadcrumb">
+        <li><a routerLink="/core/statuses">Statuses</a></li>
+        <li>{{title}}</li>
+    </ul>
```

Comment:
This is re-implementing the TP header component; it's in TPv1 I understand, but you don't need to copy this part of that page.

Reply:
I have added a heading now, is that ok ?

Reply:
No, but I submitted another review so the conversation can be had down there